File: week2/py/ayush6.py
p1 = [3, 2, 5]  
p2 = [5, 1, 2]  
max_len = max(len(p1), len(p2))
res = [0]*max_len

for i in range(max_len):
    a = p1[i] if i < len(p1) else 0
    b = p2[i] if i < len(p2) else 0
    res[i] = a + b

print("Sum Polynomial:", res)

File: week2/c/ayush6.c
#include <stdio.h>
int main() {
    int p1[3]={3,2,5};
    int p2[3]={5,1,2};
    int res[3];

    for(int i=0;i<3;i++)
        res[i]=p1[i]+p2[i];

    printf("Sum Polynomial: ");
    for(int i=0;i<3;i++)
        printf("%d ",res[i]);
    return 0;
}
